binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -241,7 +242,9 @@ public void initialize() {
             }
             onUpdate();
         } else {
-            logger.warn("Cannot connect to hue bridge. IP address or user name not set.");
+            String warn = "Cannot connect to hue bridge. IP address or user name not set.";
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, warn);
+            logger.warn(warn);
```

Comment:
> But for a general productive setup, I would assume that you rather have a file where you log ALL thing status changes, just like we currently log all item events.

On a productive setup I will log only levels that are at least warn level.

You are right, some most stuff should be handled in a more generic way. If every binding logs status changes themselves, this is not good.

So, this message needs at least logged (by the updateStatus) with at least warn severity, because it is an error.